### PR TITLE
blog: make minor corrections to upcoming SuspendJob post

### PR DESCRIPTION
**Pull request by adtac**
cc @sftim - I forgot to update https://github.com/kubernetes/website/pull/27197 before it got merged, but I agree with nearly all the review comments you posted, so here they are in a separate commit.

/sig apps

### CODE REVIEW

File: content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md [hunk 1]
```diff
@@ -55,17 +55,17 @@ spec:
 ```
 
 Jobs are not suspended by default, so I'm explicitly setting the `suspend` field
-to true in the above Job spec. In the above example, the Job controller will
+to _true_ in the above Job spec. In the above example, the Job controller will
```
Comment (adtac) [inline content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md:58]: I prefer to keep "Job spec" here to emphasise that it's in the `.spec` in addition to colloquially referring to the YAML. If you feel strongly about "Job manifest", I can change.

Comment (sftim) [reply content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md:58]: How about “the `.spec` of the Job manifest?”

**Comment by sftim** [comment]
LGTM
Will add formal review once the preview is published.

File: content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md (hunk 1 above)
Comment (adtac) [reply content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md:58]: sounds good, updated.

**Comment by sftim** [comment]
/lgtm
/approve

[Automation](https://github.com/kubernetes/website/blob/master/.github/workflows/netlify-periodic-build.yml) will publish this on the defined date.

**Pull request merged**